testing/functional/tests/cypress/plugins/index.js

```diff
@@ -15,8 +15,8 @@
 /**
  * @type {Cypress.PluginConfig}
  */
-// eslint-disable-next-line no-unused-vars
 module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
-}
+    require('cypress-mochawesome-reporter/plugin')(on);
+    // `on` is used to hook into various events Cypress emits
```

Comment:
Trivial, but template literals are used in comment.

Reply:
Added template literals.